### PR TITLE
Adding new icons for custom reports

### PR DESCRIPTION
**Pull request by jaiswalanjali**

### CODE REVIEW

File: packages/pebble-shared/svgs/unlock.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000pt" height="1600" viewBox="0 0 1000 1200"><path d="M837.5 425H340.54V232.273c0-85 71.495-154.148 159.374-154.148 87.879 0 159.375 69.148 159.375 154.148v2.102h93.75v-2.102c0-136.691-113.555-247.898-253.125-247.898-139.574 0-253.125 111.207-253.125 247.898V425H162.5c-77.54 0-140.625 63.086-140.625 140.625v478.125c0 77.54 63.086 140.629 140.625 140.629h675c77.54 0 140.625-63.09 140.625-140.629V565.625C978.125 488.085 915.039 425 837.5 425zm46.875 618.75c0 25.852-21.027 46.871-46.875 46.871h-675c-25.848 0-46.875-21.02-46.875-46.871V565.625c0-25.848 21.027-46.875 46.875-46.875h675c25.848 0 46.875 21.027 46.875 46.875zm0 0" fill="#000"/></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/unlock.svg:1]: svg clipped from top

File: packages/pebble-shared/svgs/share-with-people.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...0 49.906 40.594 90.48 90.48 90.48 49.88 0 90.473-40.574 90.473-90.48 0-49.906-40.593-90.477-90.48-90.477zm0-194.046c19.043 0 34.523 15.496 34.523 34.523s-15.48 34.523-34.523 34.523c-19.043 0-34.52-15.496-34.52-34.523s15.477-34.523 34.52-34.523zm0 0M458.332 507.144c141.805 0 257.148-115.343 257.148-257.144 0-141.8-115.343-257.144-257.148-257.144-141.8 0-257.14 115.343-257.14 257.144 0 141.8 115.335 257.144 257.14 257.144zm0-458.335c110.926 0 201.191 90.265 201.191 201.191s-90.265 201.191-201.191 201.191c-110.922 0-201.188-90.265-201.188-201.191S347.41 48.809 458.332 48.809zm0 0" stroke-width="14.285713999999999"/><path d="M759.687 663.383l.043.02c13.872 6.335 30.582.48 37.122-13.602 6.492-14.004.433-30.637-13.559-37.168l-.008-.004c-76.398-35.52-194.996-78.059-324.95-78.059-144.378 0-275.066 51.621-359.058 94.742C33.68 662.996-7.144 729.52-7.144 802.88v30.5c0 49.906 40.593 90.476 90.48 90.476H562.5c15.46 0 27.977-12.52 27.977-27.976 0-15.457-12.516-27.98-27.977-27.98H83.336c-19.043 0-34.524-15.493-34.524-34.52v-30.5c0-52.266 29.118-99.692 76.036-123.79 78.902-40.523 200.613-88.566 333.488-88.566 119.426 0 229.71 39.528 301.351 72.86zm0 0" stroke-width="14.285713999999999"/></g></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/share-with-people.svg:1]: svg clipped from sides

File: packages/pebble-shared/svgs/schedule-mail.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...7-15.73-35.294-35.176-35.294-19.453 0-35.183 15.824-35.183 35.293v23.531H710.957v-23.53c0-19.47-15.727-35.294-35.176-35.294-19.453 0-35.18 15.824-35.18 35.293v23.531H535.18v-23.53c0-19.47-15.73-35.294-35.18-35.294s-35.18 15.824-35.18 35.293v23.531H359.398v-23.53c0-19.47-15.73-35.294-35.18-35.294-19.448 0-35.179 15.824-35.179 35.293v23.531H183.617v-23.53c0-19.47-15.73-35.294-35.18-35.294-19.449 0-35.18 15.824-35.18 35.293v23.531h-83.96c-19.45 0-35.18 15.825-35.18 35.293v764.707c0 19.47 15.73 35.293 35.18 35.293h941.41c19.445 0 35.176-15.824 35.176-35.293V148.781c0-19.468-15.73-35.293-35.176-35.293zM64.477 878.195V360.547h871.046v517.648zm871.046-588.238H64.477V184.078h48.78v23.527c0 19.47 15.731 35.293 35.18 35.293 19.45 0 35.18-15.824 35.18-35.293v-23.527H289.04v23.527c0 19.47 15.73 35.293 35.18 35.293 19.449 0 35.18-15.824 35.18-35.293v-23.527H464.82v23.527c0 19.47 15.73 35.293 35.18 35.293s35.18-15.824 35.18-35.293v-23.527h105.422v23.527c0 19.47 15.726 35.293 35.18 35.293 19.448 0 35.175-15.824 35.175-35.293v-23.527h105.426v23.527c0 19.47 15.73 35.293 35.183 35.293 19.446 0 35.176-15.824 35.176-35.293v-23.527h48.781zm0 0" fill="#000" stroke-width="11.764706" stroke="#000"/></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/schedule-mail.svg:1]: `@` symbol not drawn properly

**Review by zhirzh — CHANGES_REQUESTED** [review]

File: packages/pebble-shared/svgs/share-with-people.svg (hunk 2 above)
Comment (jaiswalanjali) [reply packages/pebble-shared/svgs/share-with-people.svg]: done

File: packages/pebble-shared/svgs/schedule-mail.svg (hunk 3 above)
Comment (jaiswalanjali) [reply packages/pebble-shared/svgs/schedule-mail.svg]: for now adding it. DIscussed it with Mandeep we can modify it later.

File: packages/pebble-shared/svgs/lock.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000pt" height="1600" viewBox="0 0 1000 1200"><path d="M837.5 440.625h-84.46V247.898C753.04 111.207 639.483 0 499.913 0 360.34 0 246.79 111.207 246.79 247.898v192.727H162.5c-77.54 0-140.625 63.086-140.625 140.625v478.129C21.875 1136.922 84.961 1200 162.5 1200h675c77.54 0 140.625-63.078 140.625-140.621V581.25c0-77.54-63.086-140.625-140.625-140.625zM340.54 247.898c0-85 71.495-154.148 159.374-154.148 87.879 0 159.375 69.148 159.375 154.148v192.727H340.54zm543.835 811.48c0 25.84-21.027 46.872-46.875 46.872h-675c-25.848 0-46.875-21.031-46.875-46.871V581.25c0-25.848 21.027-46.875 46.875-46.875h675c25.848 0 46.875 21.027 46.875 46.875zm0 0" fill="#000"/></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/lock.svg:1]: `lock` icon is missing keyhole that is present in `unlock` icon

File: packages/pebble-shared/svgs/share-with-people.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...3 42.704-39.203 76.403-81.902 76.403s-78.898-33.5-81.898-76.403l-2.2-29.597C271.5 145.898 279 123.6 295 106.699c16-16.898 37.2-26.398 60.5-26.398h3.398m0-79.7h-4.5C260.2.602 185.801 80.7 192.801 174.5L195 204.102c6.102 84.597 76.602 149.796 161.602 149.796S511.699 288.5 518.199 204.102l2.2-29.602C527 80.7 452.898.602 358.898.602zM345 729.2c0-12.097 1-23.898 3.5-35.597H95.3V583.699c0-53.5 49.4-97.3 109.9-97.3h303c51.3 0 94.198 31.3 106.402 73.3l71.699-35.597c-26.2-68.204-96-117.5-178.2-117.5H205c-104.602 0-189.2 79.296-189.2 176.898v132.602c0 31.5 25.5 56.796 56.802 56.796h278.296C347 759 345 744.301 345 729.2zm0 0"/><path d="M879.3 790.5c-28.198 0-53.5 11-72.5 29.2l-173.1-80.9c.198-3.5.6-6.698.6-10.198v-1.403l175.5-86.597C828.399 656.898 852.5 667 879.5 667c57.8 0 104.602-46.8 104.602-104.602 0-57.796-46.801-104.597-104.602-104.597-57.8 0-104.602 46.8-104.602 104.597 0 2.204.204 4.903.204 6.903l-172 84.597c-19-18.796-44.903-30-73.301-30C472 623.898 425 670.7 425 728.5c0 57.8 46.8 104.602 104.602 104.602 25.5 0 48.796-9.204 67.199-24.5L775 891.8v2.898c0 57.801 46.8 104.602 104.602 104.602 57.796 0 104.597-46.801 104.597-104.602C984.102 837.5 937.301 790.5 879.301 790.5zm0 0"/></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/share-with-people.svg:1]: check if person's "head" should be circle or not

**Review by zhirzh — CHANGES_REQUESTED** [review]

File: packages/pebble-shared/svgs/lock.svg (hunk 4 above)
Comment (jaiswalanjali) [reply packages/pebble-shared/svgs/lock.svg]: Added new icons for both.

File: packages/pebble-shared/svgs/share-with-people.svg (hunk 5 above)
Comment (jaiswalanjali) [reply packages/pebble-shared/svgs/share-with-people.svg:1]: it is by designed. Discussed it with Pintu.

**Review by zhirzh — APPROVED** [review]

**Pull request merged**